tests/whitebox/monkey_patching/'

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 Red Hat, Inc.
```

Comment:
It looks like you accidentally created a file called `'`. :slightly_smiling_face: Let me know if you need any help deleting this as it can be tricky to do this in bash.